BUG: no certs directory

logs:
```go
2024-10-09T12:35:45.031+0200    INFO    config/conf-init.go:53  Successfully Created Config Manager.
2024-10-09T12:35:45.031+0200    INFO    config/conf-init.go:46  No Config found, created default Config.
2024-10-09T12:35:45.219+0200    WARN    certs/generate-certs.go:56      failed to open cert.pem for writing: open ./certs/testing.crt: no such file or directory
2024-10-09T12:35:45.219+0200    WARN    certs/generate-certs.go:70      failed to write certificate to cert.pem: invalid argument
2024-10-09T12:35:45.219+0200    WARN    certs/generate-certs.go:64      Cannot close Cert File, got: invalid argument
2024-10-09T12:35:45.219+0200    PANIC   packagelock/main.go:230 Error creating self-signed certificate: invalid argument

main.initConfig
        /home/nick/Work/programming/Golang/packagelock/main.go:230
main.init.0
        /home/nick/Work/programming/Golang/packagelock/main.go:128
runtime.doInit1
        /usr/lib/go/src/runtime/proc.go:7290
runtime.doInit
        /usr/lib/go/src/runtime/proc.go:7257
runtime.main
        /usr/lib/go/src/runtime/proc.go:254
2024-10-09T12:37:21.772+0200    INFO    packagelock/main.go:172 Successfully created Unit-File
```

Solution: 
create directory

--- a/certs/generate-certs.go
+++ b/certs/generate-certs.go
@@ -43,6 +43,11 @@ func CreateSelfSignedCert(certFile, keyFile string) error {
 		BasicConstraintsValid: true,
 	}
 
+	err = os.MkdirAll("certs/", os.ModePerm)
+	if err != nil {
+		logger.Logger.Panicf("Cannot create 'certs/' directory, got: %s", err)
+	}
+
 	// Generate a self-signed certificate using the RSA private key
 	certDER, err := x509.CreateCertificate(rand.Reader, &template, &template, &priv.PublicKey, priv)
 	if err != nil {

--- a/main.go
+++ b/main.go
@@ -74,8 +74,8 @@ var setupCmd = &cobra.Command{
 
 // Generate command
 var generateCmd = &cobra.Command{
-	Use:       "generate [certs|config|admin-user]",
-	Short:     "Generate certs or config files or an  admin-user",
+	Use:       "generate [certs|config|admin]",
+	Short:     "Generate certs or config files or an admin",
 	Long:      "Generate certificates, configuration files or an admin-user required by the application.",
 	Args:      cobra.MatchAll(cobra.ExactArgs(1), validGenerateArgs()),
 	ValidArgs: []string{"certs", "config", "admin"},
@@ -110,7 +110,7 @@ func validGenerateArgs() cobra.PositionalArgs {
 				return nil
 			}
 		}
-		return fmt.Errorf("invalid argument: '%s'. Must be one of 'certs' or 'config' or 'user'", args[0])
+		return fmt.Errorf("invalid argument: '%s'. Must be one of 'certs' or 'config' or 'admin'", args[0])
 	}
 }
 
@@ -221,10 +221,17 @@ func init() {
 		// INFO: Essential APP-Part, so crash out asap
 		panic(loggerError)
 	}
+	initConfig()
 }
 
 // generate admin for login and Setup
 func generateAdmin() error {
+	initConfig()
+	err := db.InitDB()
+	if err != nil {
+		logger.Logger.Panicf("Got error from db.InitDB: %s", err)
+	}
+
 	adminPw, err := password.Generate(64, 10, 10, false, false)
 	if err != nil {
 		logger.Logger.Panicf("Got error while generating ADmin Password: %s", err)

